func/HeroicBashLauncher.py

```diff
@@ -1,50 +1,79 @@
+#!/bin/env python3
 #Main file that takes bash arguments
 
-import os,sys
+import os, sys, argparse
 import configpath
 from frombash import frombash
 from listinstalled import listinstalled
 from steam import createscript, addtosteam
 from flatpak import launchflatpakgame
+from zenity import zenity_popup, zenity_enable
 
 #Print current version
 print("Using Bash Launcher 2.4.2\n")
 
-#Check if Zenity is installed
-print("Checking if Zenity is installed:")
-checkzenity = os.system('zenity --version')
+parser = argparse.ArgumentParser(description='Create launcher scripts from games installed with Heroic')
+parser.add_argument("--zenity", action="store_true",
+                    help="Enable zenity popups showing status as well as printing to stdout")
+group = parser.add_mutually_exclusive_group()
+group.add_argument("--create_all", action="store_true",
```

Comment:
I feel users can just `./HeroicBashLauncher` without adding the '--create_all' argument since both essentially perform the main task, i.e. to create launch scripts for all games. So cli users don't need to add this extra command.

I think that also means removing the implementation of the mutually exclusive group.